libs/sdk-ui-charts/api/sdk-ui-charts.api.md

```diff
@@ -130,12 +133,13 @@ export interface IBaseChartProps extends ICoreChartProps {
 
 // @public (undocumented)
 export interface IBubbleChartBucketProps {
-    filters?: INullableFilter[];
-    size?: IMeasure;
-    sortBy?: ISortItem[];
-    viewBy?: IAttribute;
-    xAxisMeasure?: IMeasure;
-    yAxisMeasure?: IMeasure;
+    filters?: ValuesOrPlaceholders<INullableFilter>;
+    placeholdersResolutionContext?: any;
+    size?: ValueOrPlaceholder<AnyMeasure>;
+    sortBy?: ValuesOrPlaceholders<ISortItem>;
+    viewBy?: ValueOrPlaceholder<IAttribute>;
+    xAxisMeasure?: ValueOrPlaceholder<AnyMeasure>;
```

Comment:
isn't this widening the type? before we have IMeasure only. with AnyMeasure, user can add IMeasure or IAttribute or their placeholders right?